environments/autostart.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Autostart"
+description: Learn how to configure automated environment rebuilds.
+---
+
+Coder [automatically turns off idle
+environments](../admin/environment-management/shutdown.md) to help manage
+resource expenditure. Typically, this means environments turn off overnight and
+remain offline until a rebuild is requested. With Autostart, rebuilds may be
+automatically requested at a time that best suits your workflow. You can expect
+your environments to be ready for you at the start of each work day.
+
+## Criteria for Autostart
+
+Your environment must meet the following criteria for you to use Autostart:
+
+1. You've [set the Autostart time and selected
+   environments](#enabling-autostart) to be started automatically at that time.
+
+1. Your environments are off at the Autostart time (Autostart won't work if your
+   environments are on to prevent the possibility of triggering a rebuild while
+   you're working).
+
+1. Your environment must be active. We define *inactive* environments as those
+   that haven't been opened within the last five days.
```

Comment:
The phrasing of criteria here feels a bit strange. It makes Autostart appear like a more difficult to use feature than it is IMO

Reply:
Could be adjusted to:

```md
## Triggers

Autostart triggers for environments when the following are met:

1. ...
1. ...
1. ...
```